bochs/bx_debug/parser.y

```diff
@@ -1350,6 +1351,16 @@ help_command:
          dbg_printf("    Use it as a refresh/context.\n");
          free($1);free($2);
        }
+     | BX_TOKEN_HELP BX_TOKEN_PRINT_STRING '\n'
+       {
+         dbg_printf("print-string <laddr> - prints a null-ended string from a linear address.\n");
+         free($1);free($2);
+       }
```

Comment:
it looks like 'source' is a new command (just added token) but it is not implemented yet, the token is not used outside of 'help source'

Reply:
@stlintel **source** its an old command **implemented in dbg_main.cc** (layout support use this feature) I am only adding documentation for this old command. I think its a very useful command for the user.

Reply:
@stlintel  look this: 

![oldcode](https://user-images.githubusercontent.com/9882181/189605935-fa8d02b4-a67b-4f5c-9384-14dc87e96859.png)